keyboards/nix_studio/n60a/readme.md

```diff
@@ -0,0 +1,13 @@
+# N60_A
+
+60% PCB for Sofia keyboard
+
+* Keyboard Maintainer: [Nix Studio](https://github.com/Nix-Studio) & [Valdiieee](https://github.com/valdiieee)
+* Hardware Supported: N60_A
+* Hardware Availability: Groupbuy
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```